sounds/views.py

```diff
@@ -317,15 +317,16 @@ def sound_download(request, username, sound_id):
         raise Http404
 
     if settings.FEATURE_LOG_SOUND_DOWNLOADS:
-        downloads_logger.info('Download sound', extra={
-            'user_id': request.user.id,
-            'user_ip': request.META.get('HTTP_X_FORWARDED_FOR'),
-            'protocol': request.META.get('HTTP_X_FORWARDED_PROTOCOL'),
-            'session_id': request.session.session_key,
-            'user_agent': request.META.get('HTTP_USER_AGENT'),
-            'sound_id': sound_id,
-            'range': request.META.get('HTTP_RANGE', None),
-        })
+        if 'HTTP_RANGE' not in request.META:
+            downloads_logger.info('Download sound', extra={
+                'user_id': request.user.id,
+                'user_ip': request.META.get('HTTP_X_FORWARDED_FOR'),
+                'protocol': request.META.get('HTTP_X_FORWARDED_PROTOCOL'),
+                'session_id': request.session.session_key,
+                'user_agent': request.META.get('HTTP_USER_AGENT'),
+                'sound_id': sound_id,
+                'range': request.META.get('HTTP_RANGE', None),
```

Comment:
It doesn't make sense to record this value if you are checking in the `if` block that `HTTP_RANGE` doesn't exist.

I think this check should also include if the range starts with 0, because we're not sure if it's happening on only GET requests, or if we have a HEAD to get the size of the file too.